packages/react/src/components/CommandCenter/commandCenterStore.ts

```diff
@@ -0,0 +1,87 @@
+import { IReactionDisposer, makeAutoObservable, reaction } from "mobx";
+import { InputStore } from "./InputStore";
+import { SuggestionsStore } from "./SuggestionsStore";
+import { Suggestion } from "@/constants/commandTypes";
+import { ProjectContextType } from "@/contexts/Project/useProjectContext";
+import { debounce } from "lodash";
+
+
+export class CommandCenterStore {
```

Comment:
There's a lot of business logic in here that could have been abstracted further. (Couldn't because short on time)
For example: 
- We could have a separate controller for managing the reactions and linked dependencies between the 3 stores. That'd help in separating the concerns from all 3 stores.